src/game/Accounts/AccountMgr.cpp

```diff
@@ -55,9 +55,15 @@ AccountOpResult AccountMgr::CreateAccount(std::string username, std::string pass
     const char* s_hex = srp.GetSalt().AsHexStr();
     const char* v_hex = srp.GetVerifier().AsHexStr();
 
+#ifdef DO_SQLITE
+    bool update_sv = LoginDatabase.PExecute(
+        "INSERT INTO account(username,v,s,joindate) VALUES('%s','%s','%s',datetime())",
```

Comment:
Note that `datetime()` will output UTC time, while MySQL's `NOW()` will output date and time depending on the server/connection session timezone.

Reply:
I don't think that'll be an issue because the SQLite backend doesn't mix-and-match with the MySQL backend, so it's not like we're going to have MySQL style dates in one place but SQLite style dates in another with conflicting info.
The SQLite integration is meant for quick and easy testing (swapping out the entire database by just copying a file) as well as easy portability, like our Release builds, which are currently only released for Windows because you can't just bundle a portable mysql server with a linux build. I mean you can, but try explaining that to someone who downloads the Release build because they don't know how to compile CMaNGOS

Reply:
That's just a note as I'm not sure what is expected. But I think this issue just needs to be addressed in MySQL code by switching to `UTC_TIMESTAMP()` instead of `NOW()` to avoid TZ shenanigans.

Reply:
You shouldn't migrate from SQLite to MySQL anyway. The SQLite implementation is mostly meant for testing or for being able to distribute pre-built CMaNGOS binaries that work without the hassle of setting up MySQL properly.
Of particular note is that SQLite has much fewer data-types than MySQL, so while you *can* migrate from MySQL to SQLite by squashing data types (for example all integers are 64bit in SQLite) it'll be hard to return those formats to the MySQL types. And if you're going to go through the effort of changing all times in a dumped DB to migrate it to MySQL, you might as well update the dates along with it